src/run/metrics.ts

```diff
@@ -53,13 +53,16 @@ const recursivelyGetChildren = (children: RequestsSummaryChild[]): ChildMetric[]
   );
 
 const logMetricsSummary = (summary: MetricsSummary) => {
+  const title = `Time: ${summary.date}, ${summary.duration} elapsed\n`;
   const msg =
-    `Time: ${summary.date}, ${summary.duration} elapsed\n` +
+    title +
```

Comment:
Does it duplicate the title between the group name (that we can expand) and the first line inside?

Reply:
No, here's an example:

![Capture d’écran 2023-01-04 à 10 00 44](https://user-images.githubusercontent.com/6560742/210519809-0bd93a22-8bf4-40fd-a262-51e30ba708ae.png)

Reply:
You changed it in a following commit :stuck_out_tongue: (when using logInfoGroup function)